loader/loader.c

```diff
@@ -5995,7 +6013,7 @@ VkResult setup_loader_term_phys_devs(struct loader_instance *inst) {
     // For each ICD, query the number of physical devices, and then get an
     // internal value for those physical devices.
     icd_term = inst->icd_terms;
-    for (uint32_t icd_idx = 0; NULL != icd_term; icd_term = icd_term->next, icd_idx++) {
+    for (; NULL != icd_term; icd_term = icd_term->next, icd_idx++) {
```

Comment:
Why not a while loop? Since its a more uncommon loop (with two iterands). Feel free to ignore this, its a nitpick.